### DISCUSSION TITLE
Did you forget an option value? How to pass a blank variable from yml

### DISCUSSION THREAD

**Discussion by nick5454** [cake-build/cake#3310] [General]
I have this issue, how do pass a null variable: Or is something else wrong? Upgrading to Cake 1.1.0 

Startup project is specified as a blank variable

error:
`Preparing to run build script...
Tools: D:\a\1\s\build\tools
Tools: D:\a\1\s\build\tools\Addins
Start testing Cake path.
Running build script...

Error: Expected an option value.

       .\build\build.cake --verbosity=Diagnostic --Target=Deploy 
--ZipProjects=XXXXX.Mobile.Api= --SolutionFilePath=XXXXX.Mobile.Api.sln 
--StartUpProject=
                                                                                
^ Did you forget the option value?

`

My yml:
variables:
  SYSTEM_ACCESSTOKEN: $(System.AccessToken)
  SolutionFile: "XXXXXX.Mobile.Api.sln"
  ZipProjects: "XXXXX.Mobile.Api=" # leave blank to drop in whole file
  StartUpProject: ""
  DomainProject: ""
  DomainOutputSql: ""
  DbContext: ""

stages:
- stage: GetProjectRepo
  jobs:
  - job: Build
    steps:
    - task: PowerShell@2
      displayName: 'PowerShell Script'
      inputs:
        targetType: filePath
        filePath: '.\build\build_cake.ps1'
        arguments: '--Target=Deploy --ZipProjects="$(ZipProjects)" --SolutionFilePath="$(SolutionFile)" --StartUpProject="$(StartUpProject)" --DomainProject="$(DomainProject)" --DomainOutputSql="$(DomainOutputSql)" --DbContext="$(DbContext)"'
      env:

**Comment by nils-a** [comment]
@nick5454 This has changed from pre-1.0.0 to 1.0.0 have you checked https://cakebuild.net/docs/getting-started/upgrade#passing-empty-arguments ?

**Reply by nils-a** [reply]
You could also have a look at https://github.com/cake-build/cake/discussions/3198 and/or [this answer on StackOverflow](https://stackoverflow.com/questions/66121207/how-do-you-pass-empty-arguments-in-cake-v1/66122190#66122190)

**Reply by nick5454** [reply]
Works as ZipProjects "$(ZipProjects)"
